### DISCUSSION TITLE
cmdline popup

### DISCUSSION THREAD

**Discussion by penous** [CosmicNvim/CosmicNvim#86] [Q&A] [answered]
Hey, 
Loved using CosmicNvim but I'm starting to want to learn and customize a bit more on my own. I was wondering how you created the floating popup for terminal commands triggered by `:`? It looks clean. But I can't really figure out what plugin or code does this.

Thanks!

**Comment by mattleong** [comment, marked as answer]
That would be [Noice.nvim](https://github.com/folke/noice.nvim) :)

**Reply by penous** [reply]
Thanks a lot!
